### DISCUSSION TITLE
为什么扫不到其他端口，端口每次都是固定的

### DISCUSSION THREAD

**Discussion by snyct001** [hktalent/scan4all#30] [Q&A] [answered]
比如，我本地起了Redis：6379，但是几次扫描结果都没有这个端口，对应的poc也没有执行。

**Comment by hktalent** [comment, marked as answer]
1、确定安装了nmap？
2、export PPSSWWDD=yourRootPswd
3、开启nmap
priorityNmap=true ./scan4all -host xxx.com -v -debug

只有安装nmap的前提下，Redis的指纹信息才能抓到，才能智能启动密码爆破
